openssl/src/x509/mod.rs

```diff
@@ -10,7 +10,7 @@ use std::collections::HashMap;
 use std::marker::PhantomData;
 
 use HashTypeInternals;
-use asn1::Asn1Time;
+use asn1::{Asn1Time, Asn1TimeRef};
```

Comment:
looks like `Asn1TimeRef` is causing a warning here in CI when we're not using the `x509_expiry` feature. would it be wise to wrap the `use` stmt here w/ a cfg thing? Not sure what would be idiomatic rust here or your preference.